sql/src/main/java/io/crate/operation/join/NestedLoopOperation.java

```diff
@@ -307,7 +307,7 @@ private void killBoth(Throwable throwable) {
 
         private volatile Row lastRow = null; // TODO: volatile is only required for first access
         private RepeatHandle repeatHandle;
-        private boolean wakeupRequired = true;
+        private boolean isPaused = true;
```

Comment:
no doesn't have to be volatile because it is only relevant for the left site initially - it's not used after a side-switch